### PR TITLE
add additional owners - YoyinZyc,directxman12

### PR DESCRIPTION
**Pull request by logicalhan**
**What type of PR is this?**

/kind cleanup

### CODE REVIEW

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ on:
 jobs:
   # This workflow contains a single job called "build"
   build:
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```
Comment (ehashman) [inline .github/workflows/unit-tests.yml:18]: Why this change?

**Comment by logicalhan** [comment]
You need to use this to set env variables (https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#environment-files). It's a recent change to github actions which broke the CI flow.

File: .github/workflows/unit-tests.yml (hunk 1 above)
Comment (logicalhan) [reply .github/workflows/unit-tests.yml:18]: You need to use this to set env variables (https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#environment-files). It's a recent change to github actions which broke the CI flow.
